Cache/install.sh

```diff
@@ -120,11 +123,13 @@ fi
 cp $scriptdir/cache.cpf $basedir/cache.cpf-new
 perl -pi -e 's/foia/'$instance'/g' $basedir/cache.cpf-new
 perl -pi -e 's/FOIA/'${instance^^}'/g' $basedir/cache.cpf-new
+cp $basedir/cache.cpf-new $basedir/cache.cpf
```

Comment:
This is done on line 151, was there an issue?

Reply:
It never happened for the install instance of the cache; so the post install didn't work.

Reply:
OK. I can see how that doesn't work :). carry on.